js/test/unit/generated-data-tests.ts

```diff
@@ -54,7 +54,7 @@ describe('Generated Test Data', () => {
     describe('List', () => { validateVector(generate.list()); });
     describe('Struct', () => { validateVector(generate.struct()); });
     describe('DenseUnion', () => { validateVector(generate.denseUnion()); });
-    describe('SparseUnion', () => { validateVector(generate.sparseUnion()); });
+    // describe('SparseUnion', () => { validateVector(generate.sparseUnion()); });
```

Comment:
Disabled due to https://issues.apache.org/jira/browse/ARROW-15392

Re-enabling this test case will break the JS and integration builds.

Reply:
cc @domoritz